libraries/operation-graph/src/OperationExecutionManager.ts

```diff
@@ -20,7 +20,7 @@ export interface IOperationExecutionOptions {
   parallelism: number;
   terminal: ITerminal;
 
-  requestRun?: (requestor?: string) => void;
+  requestRun?: (requester: string) => void;
```

Comment:
(The `requester` is now also non-optional)

Reply:
The requester could ultimately be multiple layers of identifiers; I guess we can just concatenate strings, but it might be better to make this an array of hierarchy nodes (e.g. Operation, file). When Rush receives one of these messages over IPC it'll get wrapped in a further node for the Rush-level Operation.